packages/admin/src/components/Identity/IdentityProvider.tsx

```diff
@@ -155,17 +155,15 @@ export const IdentityProvider: React.FC<IdentityProviderProps> = ({ children, on
 		return <InvalidIdentityFallback />
 	}
 
-	if (identityState.state === 'loading' || (!allowUnauthenticated && identityState.state === 'none')) {
-		return <ContainerSpinner />
-	}
-
 	return (
-		<EnvironmentExtensionProvider extension={identityEnvironmentExtension} state={identityContextValue?.identity ?? null}>
-			<IdentityContext.Provider value={identityContextValue}>
-				<IdentityRefreshContext.Provider value={refetch}>
-					{children}
-				</IdentityRefreshContext.Provider>
-			</IdentityContext.Provider>
-		</EnvironmentExtensionProvider>
+		<SpinnerContainer enabled={identityState.state === 'loading' || (!allowUnauthenticated && identityState.state === 'none')}>
```

Comment:
this may imho break some apps, which may require a initialized idnetity

Reply:
Done